src/content/blog/astro-240.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: "Astro 2.4"
+description: ""
+publishDate: "May 4, 2023"
+authors:
+  - matthew
+coverImage: "/src/content/blog/_images/astro-220/blog-hero-2_2-release.jpg"
+socialImage: "/src/content/blog/_images/astro-220/og-image-2_2-release.jpg"
+lang: "en"
+---
+
+Astro 2.4 just dropped, and features:
+
+- **[Stronger scoping for CSS](#scopedstylestrategy)**: A new opt-in flag `scopedStyleStrategy` allows you to configure your Astro components to use a high specificity for its styles.
+- **[Improved &lt;Code /&gt; component](#code--component-improvements)**: Upgraded Shiki and support for inlining.
+- **[SSR support in sitemaps](#ssr-in-sitemaps)**: Known routes are now built into a sitemap during the build.
+- **[Middleware (experimental)](#middleware-experimental)**: Define code that runs before page components and endpoints, to intercept requests and modify responses.
+- **[CSS inlining (experimental)]()**: Allows you to configure when styles should be inlined via a `<style>` tag.
+
+## scopedStyleStrategy
+
+Start in 1.0 Astro uses the `:where` pseudo-selector for scoped Astro component styles. This approach means your styles have the same specificity as if they were written in plain CSS. The downside of this approach is that you need to be conscience of the selectors used in components.
```

Comment:
```suggestion
Start in 1.0 Astro uses the `:where` pseudo-selector for scoped Astro component styles. This approach means your styles have the same specificity as if they were written in plain CSS. The downside of this approach is that you need to be conscious of the selectors used in components.
```

I think this is a typo? 

I also think the *"Start in 1.0 Astro uses the `:where` pseudo-selector for scoped Astro component styles."* is a bit hard to read?

Reply:
Yes! Went with Sarah's recommendation here. Thanks for spotting.